sql1.txt

```diff
@@ -32,12 +52,25 @@ SELECT CustomerName FROM Customers WHERE CustomerID NOT IN (SELECT CustomerID FR
 
 This is a subquery -- we haven't talked about those.  But, you can do the same thing with a left
 join by comparing the OrderID column with NULL.  Paste your SQL statement below.
+                             
+                              - SELECT c.CustomerName, o.OrderID
+                                FROM Customers c
+                                LEFT JOIN Orders o
+                                ON c.CustomerID = o.CustomerID
+                                WHERE o.OrderID IS NULL;
 
 
 
 7. Retrieve the OrderID and ProductName of every order for a product with Tofu in the name.
 Paste your SQL statement below.  You will have to join Orders, OrderDetails, and Products.
-
+                             
+                              - SELECT o.OrderID, p.ProductName
+                                FROM Orders o 
+                                JOIN OrderDetails od
+                                ON o.OrderID = od.OrderID
+                                JOIN Products p
+                                ON od.ProductID = p.ProductID
+                                WHERE p.ProductName LIKE "%tofu%";
 
 
```

Comment:
nice! - 
keep your eyes on case consistency: 
Ensure case consistency in the LIKE clause for `Tofu`